sigopt/xgboost/compat.py

```diff
@@ -0,0 +1,37 @@
+# pylint: disable=unused-import
+MIN_REQUIRED_XGBOOST_VERSION = "1.3.0"
+
+
+try:
+  import xgboost
+  from xgboost import Booster, DMatrix
+except ImportError as ie_xgb:
+  raise ImportError(
+    "xgboost needs to be installed in order to use sigopt.xgboost.run functionality. "
+    "Try running `pip install sigopt [xgboost]`."
```

Comment:
I think you'll want `pip install 'sigopt[xgboost]'`. I think pip will try to install the package `"[xgboost]"`, which does not exist. Also, some shells treat `[]` as special characters so they need to be in quotes.